archivy/cli.py

```diff
@@ -22,20 +26,77 @@ def cli():
 
 
 # add built in commands:
-cli.add_command(routes_command)
 cli.add_command(shell_command)
 
 
+@cli.command("init", short_help="Initialise your archivy application")
+@click.pass_context
+def init(ctx):
+    try:
+        load_config()
+        click.confirm("Config already found. Do you wish to reset it? "
+                      "Otherwise run `archivy config`", abort=True)
+    except FileNotFoundError:
+        pass
+
+    config = Config()
+    delattr(config, "SECRET_KEY")
+
+    click.echo("This is the archivy installation initialization wizard.")
+    set_curr_dir = click.confirm("Use current directory as data directory for archivy?")
+    if set_curr_dir:
+        data_dir = os.getcwd()
+    else:
+        data_dir = click.prompt("Otherwise, enter the full path of the "
+                                "directory where you'd like us to store data.", type=str)
+
+    es_enabled = click.confirm("Would you like to enable Elasticsearch? For this to work "
+                               "when you run archivy, you must have ES installed.")
+    if es_enabled:
+        config.ELASTICSEARCH_CONF["enabled"] = 1
+    else:
+        delattr(config, "ELASTICSEARCH_CONF")
+
+    create_new_user = click.confirm("Would you like to create a new admin user?")
+    if create_new_user:
+        username = click.prompt("Username")
+        password = click.prompt("Password", hide_input=True, confirmation_prompt=True)
+        if not ctx.invoke(create_admin, username=username, password=password):
+            return
+
+    try:
+        pypandoc.get_pandoc_version()
+    except OSError:
+        download_pandoc = click.confirm("Archivy requires Pandoc to be installed. "
+                                        "Do you want us to install it automatically?")
+        if download_pandoc:
+            pypandoc.download_pandoc()
+
+    config.override({"USER_DIR": data_dir})
+    app.config["USER_DIR"] = data_dir
+
+    # create data dir
+    os.makedirs(os.path.join(data_dir, "data"), exist_ok=True)
+
+    write_config(vars(config))
+    click.echo("Config successfully created at "
+               + os.path.join(app.config['INTERNAL_DIR'], 'config.yml'))
+
+
+@cli.command("config", short_help="Open archivy config.")
+def config():
+    open_file(os.path.join(app.config["INTERNAL_DIR"], "config.yml"))
+
+
 @cli.command("run", short_help="Runs archivy web application")
 def run():
     click.echo('Running archivy...')
     load_dotenv()
     watcher = Watcher(app)
     watcher.start()
-    port = int(os.environ.get("ARCHIVY_PORT", 5000))
     os.environ["FLASK_RUN_FROM_CLI"] = "false"
     app_with_cli = create_click_web_app(click, cli, app)
-    app_with_cli.run(host='0.0.0.0', port=port)
+    app_with_cli.run(host='0.0.0.0', port=app.config["PORT"])
```

Comment:
Why is it dangerous?

Reply:
Many people do not know what they are actually doing. see https://superuser.com/questions/1013740/is-it-bad-practice-to-listen-on-0-0-0-0-why; archivy is not _always_ meant to be hosted on a server. Or, do you not want users to use it locally? at `localhost`?

Reply:
`0.0.0.0` would also work on localhost but yes I see what you mean, I'll add it as a config with `0.0.0.0` as a default for more flexibility.

Reply:
No, I don't agree. It should be 127.0.0.1 by default. Maybe ask a question to the user in the configuration wizard:
```
Where do you plan to host archivy 
(i) only on your computer (localhost)
(ii) on your local network (0.0.0.0)
>>>
```
Hosting on `0.0.0.0` loses privacy sometimes, even on local network..., it could even be a public wifi, you dont want your data to be exposed, even if you have an admin login system. It is better to let the users know of it explicitly

Reply:
Oh yeah I see what you mean, ok I'll do that.